crates/mdbook-mun-examples-test/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "mdbook-mun-examples-test"
```

Comment:
```suggestion
name = "mun_mdbook_test"
```

Reply:
To be consistent with our other crate names. @baszalmstra any other ideas for naming?

Reply:
`mun_mdbook_tester`?

Reply:
`mun_mdbook_test_plugin`?